src/styles/blocks/header.scss

```diff
@@ -0,0 +1,51 @@
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  .header__image {
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: $nav-link-color;
+  padding: 23px 0;
+  display: inline-block;
+  font-weight: 500;
+}
+
+.is-active,
+.nav__link:hover {
+  color: $card-button-color;
+}
+
+.nav__item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.is-active::after {
```

Comment:
create modifier for the links and move these styles to modifier
use is-active class only in html for tests passing